solidity/test/hooks/ProtcolFee.t.sol

```diff
@@ -0,0 +1,96 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.13;
+
+import "forge-std/Test.sol";
+
+import {StaticProtocolFee} from "../../contracts/hooks/StaticProtocolFee.sol";
+
+contract StaticProtocolFeeTest is Test {
+    StaticProtocolFee internal fees;
+
+    address internal alice = address(0x1);
+
+    function setUp() public {
+        fees = new StaticProtocolFee(1e15, address(this));
+    }
+
+    function testConstructor() public {
+        assertEq(fees.protocolFee(), 1e15);
+    }
+
+    function testSetProtocolFee(uint256 fee) public {
+        fee = bound(fee, 0, fees.MAX_PROTOCOL_FEE());
+        fees.setProtocolFee(fee);
+        assertEq(fees.protocolFee(), fee);
+    }
+
+    function testFuzz_postDispatch_inusfficientFees(
+        uint256 feeRequired,
+        uint256 feeSent
+    ) public {
+        feeRequired = bound(feeRequired, 1, fees.MAX_PROTOCOL_FEE());
+        // bound feeSent to be less than feeRequired
+        feeSent = bound(feeSent, 0, feeRequired - 1);
+        vm.deal(alice, feeSent);
```

Comment:
this is a bit confusing

Reply:
what's confusing here, the feeSent < feeRequired for check revert here